content/docs/1.6.0/advanced-resources/backing-image-backup.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Backing Image Backup
+weight: 4
+---
+
+Longhorn supports backing up backing images since v1.6.0.
+
+> **Prerequisite:** A backup target must be set up. For more information, see [Set the BackupTarget](../snapshots-and-backups/backup-and-restore/set-backup-target). If the BackupTarget has not been set, you'll be presented with an error.
+
+
+## Create and Restore Backing Image Backup
+
+Since Backing Image is globally unique in Lnoghorn system, its Backup is also globally unique with the same name.
+
+### Create a backing image backup via YAML
+
+You can create a backing image backup by YAML. Here is the example:
+
+Assuming you already have one backing image as follow:
+```yaml
+apiVersion: longhorn.io/v1beta2
+kind: BackingImage
+metadata:
+  name: parrot
+  namespace: longhorn-system
+spec:
+  sourceType: download
+  sourceParameters:
+    url: https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw
+  checksum: 304f3ed30ca6878e9056ee6f1b02b328239f0d0c2c1272840998212f9734b196371560b3b939037e4f4c2884ce457c2cbc9f0621f4f5d1ca983983c8cdf8cd9a
+```
+
+You can then create the backup for this backing image by applying the YAML
+```yaml
+apiVersion: longhorn.io/v1beta2
+kind: BackupBackingImage
+metadata:
+  name: parrot
+  namespace: longhorn-system
+spec:
+  userCreated: true
+  labels:
+    usecase: test
+    type: raw
+```
+
+- `name`: should be the same as the backing image or Longhorn won't be able to find the backing image.
+- `userCreated`: indicates whether this Backup CR is created by user to create the backup in backupstore or it is synced from backupstrore.
+- `labels`: just like volume backup, you can add labels to the backing image backup.
+
+### Restore a backing image from backupstore via YAML
+
+After creating a backup in the backupstore, you are now be able to restore it back in another cluster. Here is the example:
+
+```yaml
+apiVersion: longhorn.io/v1beta2
+kind: BackingImage
+metadata:
+    name: parrot-restore
+    namespace: longhorn-system
+spec:
+    sourceType: restore
+    sourceParameters:
+        # change to your backup URL
+        # backup-url: nfs://longhorn-test-nfs-svc.default:/opt/backupstore?backingImage=parrot
+        backup-url: s3://backupbucket@us-east-1/?backingImage=parrot
+        concurrent-limit: "2"
```

Comment:
Where are the credentials?

Reply:
user don't need to provide the credentials
when Longhorn do the bacuup
it will get the backupTarget and the secret automatically and use that credentials
you can see the code here: https://github.com/longhorn/longhorn-manager/pull/2416/files#diff-3ddec43f26f529baed3c3d660a194662725352c76e4e734d6f7ffbb51325e574R686

its the same as backup volume , user also don't need to provide the credentials to the CR when backup and restore.